tests/components/form/form-input.test.tsx

```
@@ -0,0 +1,580 @@
+
+// Mock react-hook-form
+vi.mock('react-hook-form', () => ({
+	useFormContext: vi.fn(),
+}))
+
+// Mock UI components
+vi.mock('@/components/ui', () => ({
+	Input: ({ children, ...props }: any) => (
+		<input data-testid='form-input' {...props}>
+			{children}
+		</input>
```

Biome (2.1.2)

[error] input is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)